### PR TITLE
Admin able to delete not approved users

### PR DESCRIPTION
**Pull request by DimaSmagulov**
code review:
- [ ] @CHILL99 
- [ ] @krisberry 
- [x] @natalya5 
- [x] @oleg08 
- [x] @OrestF 
- [x] @rtershak 
- [x] @vvs79 

2nd lvl review:

- [x] @Kmalynss
- [ ] @serhiym 
- [ ] @ruba-ruba

There is no such task, but i think it will be usefull.
1. Admin able to delete not approved users.
2. Added length validation for questions and topic models.
3. Fixed questions and topic tests.
4. Added welcome message to admin dashboard.
5. Added close buttons to Activerecord arrors.
6. Some small style and bug fixes.
7. Added users seeds. (Didn't add pagination for user#index, coz saw that my teammate has allready done that in his PR)

### CODE REVIEW

**Comment by OrestF** [comment]
reviewed

**Comment by oleg08** [comment]
reviewed

**Comment by vvs79** [comment]
reviewed

**Comment by natalya5** [comment]
reviewed

**Comment by rtershak** [comment]
reviewed

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -15,6 +16,12 @@ def approve
     redirect_to admin_users_path, notice: 'User not found'
   end
 
+  def destroy
+    @user = User.find(params[:id]).destroy
+      flash[:success] = 'User deleted'
```
Comment (Kmalynss) [inline app/controllers/admin/users_controller.rb]: please handle situation when there is no user with provided id

Comment (DimaSmagulov) [inline app/controllers/admin/users_controller.rb]: Added rescue ActiveRecord::RecordNotFound

File: app/views/admin/users/_user.html.slim [hunk 2]
```diff
@@ -6,7 +6,14 @@ tr.info
...
+      i.glyphicon.glyphicon-ok.text-success aria-hidden="true" 
+    - else 
+      = link_to approve_admin_user_path(user), class: 'btn btn-sm btn-success', rel: 'tooltip', title: 'Approve user'
+        i.glyphicon.glyphicon-ok
+      = link_to admin_user_path(user), method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-sm btn-danger', rel: 'tooltip', title: 'Delete user'
+        i.glyphicon.glyphicon-remove
   td
-    = user.admin? ? (raw "&#10004;") : ''
+    - if user.admin?
+      i.glyphicon.glyphicon-ok.text-success aria-hidden="true"
```
Comment (OrestF) [inline app/views/admin/users/_user.html.slim]: add new line here

File: app/views/layouts/_header.html.slim [hunk 3]
```diff
@@ -22,6 +22,6 @@ nav.navbar.navbar-default.navbar-fixed-top.topnav[role="navigation"]
         - else
           li
             = link_to('Login', new_user_session_path)
-- if user_signed_in?
-  .navbar-brand.topnav.breadcrumb
-    = render_breadcrumbs separator: " > "
+  - if user_signed_in?
+        .breadcrumb.container
+          = render_breadcrumbs separator: " > "
```
Comment (OrestF) [inline app/views/layouts/_header.html.slim]: new line

File: test/controllers/users_controller_test.rb [hunk 4]
```diff
@@ -6,7 +6,7 @@ class UsersControllerTest < ActionController::TestCase
     sign_in create(:admin)
     get :show
     assert_response :success
-    assert_select "h2", 'Admin dashboard'
+    assert_select "h2", 'Welcome admin!'
   end
```
Comment (OrestF) [inline test/controllers/users_controller_test.rb]: I'm not sure, but maybe previous variant is better.

**Comment by Kmalynss** [comment]
gtg

**Pull request merged**